elastic/kibana
# [Security Solution][Endpoint] Add responder `scan` action

## Summary
Adds `scan` response action to Responder.

### Test
1. Add `responseActionScanEnabled` feature flag to `xpack.securitySolution.enableExperimental`
2. Run ES/Kibana
3. Run `node x-pack/plugins/security_solution/scripts/endpoint/run_endpoint_agent.js` to start a VM with Elastic Defend installed.
4. Visit `app/security/administration/endpoints` and click on the endpoint on the endpoint list.
5. Click on the action kebab menu and open responder. 
6. Click on `Help` to verify that the `scan` action is available for the endpoint and doesn't have an alert icon next to it.
7. Click on the `+` icon next to `scan` help command to add it to the input and provide a file path from your VM to scan. Send the request by clicking the `play` icon or hitting the `return` key.
8. You should see the scan action being carried out and eventually resulting in either a `Scan complete` or error info.

![Screenshot 2024-06-04 at 11 52 32](https://github.com/elastic/kibana/assets/1849116/4a1a42e5-85d0-407f-962c-7ee40951417e)

![Screenshot 2024-06-04 at 11 58 12](https://github.com/elastic/kibana/assets/1849116/b5911473-f000-4cee-83b4-c79166829d39)

![Screenshot 2024-06-05 at 12 04 59](https://github.com/elastic/kibana/assets/1849116/950d79ed-e69d-4a3c-8bcb-a38fc3d9a91e)

### clip
![scan-command](https://github.com/elastic/kibana/assets/1849116/331deabc-faf4-44e8-8248-e7b86f2add99)


### Checklist
